src/snowflake/snowpark/udf.py

```diff
@@ -284,9 +284,9 @@ class UDFRegistration:
             >>> @udf(packages=["numpy"])
             ... def sin_udf(x: float) -> float:
             ...     return np.sin(x)
-            >>> df = session.create_dataframe([0.25 * math.pi, 0.5 * math.pi], schema=["d"])
+            >>> df = session.create_dataframe([0.0, 0.5 * math.pi], schema=["d"])
             >>> df.select(sin_udf("d")).to_df("col1").collect()
-            [Row(COL1=0.7071067811865475), Row(COL1=1.0)]
```

Comment:
This one is flaky due to arithmetic precision, so after this bug is fixed, we can use 0 directly to avoid flakiness.